baksnapperd.sh

```diff
@@ -125,17 +126,16 @@ case "$1" in
     link-latest)
         shift
         declare -a snapshots
-        for dir in "$1"/*; do
+        if [[ -h "$1/latest" ]]; then
```

Comment:
I think I discovered a bug (not introduced by this PR). If "$1/latest" exist but is not a symlink it will not remove it. Which is good, but on line 138 it will blindly try and create a symlink regardless if the path exist or not. Worst case it will delete the latest file or create a symlink inside it if it's a directory.

Better to error out if it's not a symlin

E.g.
```
if [[ -h "$1/latest" ]]
then
    rm "$1/latest"
else
    error "latest is not a symlink"
fi
```